xml/System.Net.Http/HttpClient.xml

```diff
@@ -424,10 +451,23 @@ public class GoodController : ApiController
         <ReturnType>System.Version</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
-      </Docs>
+        <summary>Gets or sets the default version used on subsequent requests made by this <see cref="T:System.Net.Http.HttpClient" /> instance.</summary>
+        <value>The <see cref="P:System.Net.Http.HttpRequestMessage.Version" /> which is going to be set on <see cref="T:System.Net.Http.HttpRequestMessage" /> instances created by this <see cref="T:System.Net.Http.HttpClient" /> instance.</value>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+<xref:System.Net.Http.HttpClient.DefaultRequestVersion%2A> is <xref:System.Net.HttpVersion.Version11> by default.
+
+The <xref:System.Net.Http.HttpClient.DefaultRequestVersion%2A> property would be set as the <xref:System.Net.Http.HttpRequestMessage.Version%2A> on any <xref:System.Net.Http.HttpRequestMessage> the <xref:System.Net.Http.HttpClient> instance creates, in example, as part of a call to <xref:System.Net.Http.HttpClient.GetAsync%2A>, <xref:System.Net.Http.HttpClient.GetStringAsync%2A>, <xref:System.Net.Http.HttpClient.GetByteArrayAsync%2A>, etc.
+
+The <xref:System.Net.Http.HttpClient.DefaultRequestVersion%2A> property can be overriden as long as the <xref:System.Net.Http.HttpClient> instance has not started any request.
```

Comment:
```suggestion
The `DefaultRequestVersion` property can be overriden as long as the <xref:System.Net.Http.HttpClient> instance has not started any request.
```
Do you mean overridden (as in a derived class overrides the DefaultRequestVersion property) or can be changed?

Reply:
I meant "can be changed". thanks for catching that.